src/data/matrix/kronecker.lean

```diff
@@ -221,7 +233,10 @@ lemma mul_kronecker_mul [comm_semiring α]
   (A ⬝ B) ⊗ₖ (A' ⬝ B') = (A ⊗ₖ A') ⬝ (B ⊗ₖ B') :=
 kronecker_map_linear_mul_mul (algebra.lmul ℕ α).to_linear_map mul_mul_mul_comm A B A' B'
 
--- insert lemmas specific to `kronecker` below this line
+lemma kronecker_assoc [comm_semiring α] {q r : Type*} [fintype q] [fintype r]
```

Comment:
Can you move `q` and `r` to the top of the file? Or alternatively, use one of `l' m' n' p'`. I don't really mind which.

Reply:
Sure, but don't they get implicit variables for all the intermediate lemmas?

Reply:
No, they only end up in lemmas that reference them.

Reply:
Ah ok, then I implement this.

Reply:
Done, I had to create a new line to stick to the 100 chars.